### ISSUE TITLE
Feature Request: RDAdmin -> Qt Style selector?

### ISSUE THREAD

**Issue by alexolivan** [ElvishArtisan/rivendell#733]
Hi!

Today's build added an interesting feature... the --show-styles option to all modules, which, to me, is very very welcome!
Using the -style= option on 3.X/Qt4 modules was (is) very much needed to achieve dark themeing. 
However, at the very end, relying on env variables or tweaking launchers to append --style option to override defaults, can be tedious, and may not always work (for example, some modules in 3.X such as rdlogmanager, behave as qtstyle-hardcoded)

Now, with Qt5, everything is somehow far more customizable (complex?) on the operating system level, with more and more options and GUI tools to customize look and feel (gtk2/lxappearance, Qt5/qt5ct , Kvantum Qt5 theme GUI manager, etc, etc...).
But it all depends, at the very end, on the style the module is launched as:

plastique, windows, whatever....
gtk2 -> it follows OS/GTK themeing and settings
qt5ct-style -> it follows OS/QT5 themeing and settings 
kvantum/kvantum-dark - it follows the Qt5 Kvantum theme settings

It would be nice to have, in RDAdmin, at host configuration, a way to choose a default style from the detected available ones (like --show-styles code achieves), so Qt style for modules at launch becomes both controllable and a certainty.

Regards!

**Comment by ElvishArtisan** [comment]
Yes, it can be fun try out all of the different styles. Unfortunately however, not all of them work well with Rivendell. For example, the 'Motif' style (included as one of the standard set by Qt) does not work well at all, mostly because the widgets expect more room to render text than Rivendell is laid out to provide. The 'Adwaita' and 
'Adwaita-Dark' styles that RHEL-8 provides as the default suffers similarly (although it is otherwise much more pleasing to the eye than Motif).

For this reason, I've always regarded running Rivendell with a non-default style to be more of a hack than anything. One can do it, and very occasionally it can be necessary, but it's not something I really want to encourage. I've spent the last couple days trying out the various styles on various distros, and have come to the conclusion that the best for Rivendell's purposes remains 'plastique', which (if you don't count a couple of name changes over the years) has been the Rivendell default for nearly two decades.
